webull/webull.py

```diff
@@ -89,23 +89,43 @@ def login(self, username='', password=''):
         except EmailNotValidError as _e:
           accountType = 1 # phone
 
+        if device_name == '' :
+            device_name = 'default_string'
+
         data = {
             'account': username,
             'accountType': accountType,
             'deviceId': self._did,
+            'deviceName': device_name,
+            'extInfo': {'verificationCode': mfa},
```

Comment:
Should this be sent with every login attempt or only when an unknown device is required to get an access code?

Reply:
you are right, it would be better if this is only submitted at mfa requests.